### PR TITLE
Add London to Master

### LINKED ISSUE #38: vicuk - London
## London

- [x] Create a branch
- [x] Add file
- [x] Create a commit
- [x] Open a PR
- [ ] Have a discussion about PR
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by vicuk**
Closes #38

### CODE REVIEW

File: places/vicuk-london.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## London
+
+### Fun Facts
```
Comment (larryeh) [inline places/vicuk-london.md]: Do you like Blur?  They are one of my favorite bands...

Comment (jibralta) [inline places/vicuk-london.md]: Can you tell me a fun fact?  I've never been to London.  Hope I commented correctly. :)

Comment (dschz) [inline places/vicuk-london.md]: @jibralta Big Ben is the bell, not the clock tower. Its chime is in the key of E.

**Comment by vicuk** [comment]
@jibralta Yes, London has the world's oldest subway. I added it to the file.

**Comment by vicuk** [comment]
@larryeh I don't know Blur, I will look for them.

**Comment by vicuk** [comment]
@dschz I didn't say Big Ben was the tower. Were you responding to something someone wrote?

**Comment by Cohen611** [comment]
London rocks!

File: places/vicuk-london.md (hunk 1 above)
Comment (Cohen611) [inline places/vicuk-london.md]: This is a test comment.

**Comment by kha3474** [comment]
good to go

**Pull request merged**